VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+
+If you'd like to request removal from the list, or an update to your policy entry (or associated contact email), contact us at [starttls-policy@eff.org](mailto:starttls-policy@eff.org)
+
+## Threat modelling and security considerations
+
+### DNS forgery
+
+An attacker with the ability to forge DNS requests to a sending mailserver for a particular receiving mailserver might provide fake MX records for the receiving mailserver, causing the sender to direct its mail towards a malicious server. This is thwarted if the sender validates against the policy list and discovers that the receiving server uses a different set of hostnames.
+
+This type of attacker may try and induce a removal and re-addition of a policy to the list. To do this, they might need to induce a validating CA to misissue a certificate, or maintain a machine-in-the-middle position on our validation server for several weeks, and against our notification email server, which will send updates to the policy's email contact (also unknown to the attacker, and could be hosted on a different server) about these attempts.
+
+### TCP on-path attacker
+
+Normally, a regular on-path attacker might simply downgrade TLS. A more sophisticated one might perform a certificate man-in-the-middle. In either case, this is thwarted if the sender is validating against the policy list and discovers that the receiving server should support TLS with a valid certificate.
+
+This type of attacker may try and induce removal of that receiving server from the policy list. To prevent this, we'll issue a DNS challenge to validate any policy list removal.
+
+### CA compromise or induced misissuance
+
+When a valid MTA-STS record is discovered, the STARTTLS policy list cannot necessarily provide a higher degree of security, since we also validate TLS policies from our own network position. Thus, MTA-STS policies should take precedence over the STARTTLS policy list. 
+
+This means that we do assume CA compromise or misissuance is more difficult (in terms of cost, risk, and detectability thanks to CT) than performing a single DNS on-path attack or forgery, and than performing a TCP on-path attack. However, it is possible, with a temporary DNS machine-in-the-middle position (perhaps due to BGP hijacking) with respect to an automated CA like Let's Encrypt, to induce certificate misissuance. 
+
+Some of these judgments depend on an adversary's relative network perspective to the target mailserver and various automated CAs. Regardless, inducing misissuance is at least detectable if a domain owner is monitoring CT logs, and may require extra effort (depending on network perspective) in addition to having a machine-in-the-middle DNS position targeted towards a particular mailserver. Hopefully, with [multi-perspective validation](https://letsencrypt.org/upcoming-features/) on the horizon for Let's Encrypt, the bar could be raised even higher for an attacker to induce misissuance.
```

Comment:
Is there a good page somewhere that describes how to monitor CT logs? This might a good idea for most server admins and also something that they're typically not familiar with yet.

Reply:
Not that I know of. Would it be helpful for us to publish something of the sort? (i.e. what is CT, what does it mean to "monitor" the logs)

For now i'll link to https://www.certificate-transparency.org/ !